release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1151,6 +1159,12 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 [id="ocp-4-10-storage-bug-fixes"]
 ==== Storage
 
+* Previously, Local Storage Operator (LSO) took an unacceptably long time to delete orphaned persistent volumes (PVs) due to  the accumulation of a 10-second delay. Now LSO does not use the 10-second delay, and thus PVs are deleted promptly, and  local disks are made available for new persistent volume claims (PVCs) much more quickly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2001605[*BZ#2001605*])
```

Comment:
looks like an extra space after "due to"

Reply:
With this update, LSO does not use the 10-second delay, PVs are deleted promptly, and local disks are made available